services/web/server/tests/unit/with_dbs/config-devel.yml

```diff
@@ -42,7 +42,6 @@ rabbit:
     celery:
       result_backend: rpc://
     log: comp.backend.channels.log
-    progress: comp.backend.channels.progress
```

Comment:
Was this the channel used to publish the progress of the computational service to the container? I must have missed how with what it was replaced.

Reply:
this channel was actually never used. the progress is sent back through the logs channel.